src/views/Actors/useActors.tsx

```diff
@@ -380,10 +351,39 @@ export const useActors = (actors: Team[], stories = false) => {
       ),
 
       style: {
-        width: 165,
+        width: 133,
         menuWidth: 350,
       },
     },
+    {
+      type: 'select',
+      id: 'actor_role',
+      label: 'Actor Role',
+      selected: filteredCategories,
+      multiple: true,
+
+      options: categoryOptions,
+      onChange: (value: string | number | (string | number)[]) => {
+        if (typeof value === 'string' || (Array.isArray(value) && value.every((item) => typeof item === 'string'))) {
+          handleChangeUrl('filteredCategories')(value as string | string[]);
+        } else {
+          console.error('Invalid value type: must be string or string array');
```

Comment:
url params are string or arrays only, and you're already checking that, if the values are any other type, it is a programming error so we don't need to handle this console. It doesn't make too much sense from the user perspective anyway as it is a console print

Reply:
done